src/WMBot/Commands/Channel.cs

```diff
@@ -167,6 +167,7 @@ public static void PartChannel(Channel channel, string user, string host, string
                         lock (Configuration.Channels)
                         {
                             channel.Remove();
+                            IRC.DeliverMessage("Successfully parted " + channel_name + ".");
```

Comment:
OK, this is a definition of function I recommend to use (IRC/IRC.cs):

```
        /// <summary>
        /// Send a message to a given target
        /// </summary>
        /// <param name="text">Text of a message</param>
        /// <param name="target">Channel</param>
        /// <param name="priority">Priority</param>
        public static void DeliverMessage(string text, Channel target, libirc.Defs.Priority priority = libirc.Defs.Priority.Normal)
        {
            // body trimmed
        }
```

You can see it's had 3 arguments, last one optional. You provided only 1, so this wouldn't compile. Also I don't see a point in sending this message in case that @part was executed from the channel that is about to be parted.

The correct version of code would probably look like this:

```
lock (Configuration.Channels)
{
    channel.Remove();
    if (origin != channel_name)
        IRC.DeliverMessage("Successfully parted " + channel_name, origin);
```

The trailing dot would cause troubles as it's a valid symbol for name and some irc clients would display broken channel name (they would point you to #meh. instead of #meh)